src/integrationTest/resources/Tests/CourthousesTest/courthousesPutEndpoint/requestBodySwansea.json

```diff
@@ -0,0 +1,4 @@
+{
+  "courthouse_name" : "SWANSEA",
+  "code" : "457"
```

Comment:
Does this need to be changed to be
`"code" : 457` ?

Reply:
ACCEPT